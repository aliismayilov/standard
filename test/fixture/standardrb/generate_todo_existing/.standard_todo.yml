# Auto generated files with errors to ignore.
# Remove from this list as you refactor files.
---
ignore:
- useless_assignment.rb:
  - Lint/UselessAssignment
