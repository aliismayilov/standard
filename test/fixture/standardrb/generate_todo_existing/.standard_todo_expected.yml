# Auto generated files with errors to ignore.
# Remove from this list as you refactor files.
---
ignore:
- assignment_in_condition.rb:
  - Lint/AssignmentInCondition
- useless_assignment.rb:
  - Lint/UselessAssignment
